### PR TITLE
Adding info notes to ACAS form.

### PR DESCRIPTION
**Pull request by zaparka**
To reduce incorrectly checked in "I don't have an Acas number" we are adding more information to the form.

### CODE REVIEW

File: app/assets/javascripts/modules/moj.reveal-hints.js [hunk 1]
```diff
@@ -11,4 +11,20 @@
...
     }
   });
+})();
+
+(function() {
+  $('.panel-indent input[type=radio]').each(function(i, field){
+    var container = $(field);
+    if(container.length){
+      container.on('change', function(){
+        var hint = $(this).parents('.panel-indent').children('.reveal-acas-hint');
```
Comment (garytaylor) [inline app/assets/javascripts/modules/moj.reveal-hints.js]: is this right ?  I thought this was an mot module which is global - but it has something about 'acas hints' in it ?

Comment (zaparka) [reply app/assets/javascripts/modules/moj.reveal-hints.js]: @garytaylor this is a module that reveals and hide all hints, but I can't use it. The view is using collection so I can't dynamically assign data attribute based on the checkbox value. So I thought that this is the best place to put it. I can use a new file, but It makes sense to me to keep it together with similar functionality. But if you would insist on putting into another file, I don't mind.

Comment (garytaylor) [reply app/assets/javascripts/modules/moj.reveal-hints.js]: No not at all - just wanted to check it wasn't a mistake thats all - if you're happy with it in there so am I .

Comment (zaparka) [reply app/assets/javascripts/modules/moj.reveal-hints.js]: cool thx

**Pull request merged**